### ISSUE TITLE
Heroku Error Log [1.1.2]

### ISSUE THREAD

**Issue by bonxz** [GiorgioBrux/nitro-sniper-enhanced#16]
```
2021-01-09T19:06:24.996622+00:00 app[worker.1]:     at WebSocketPacketManager.handleQueue (/app/node_modules/discord.js/src/client/websocket/packets/WebSocketPacketManager.js:72:16)
2021-01-09T19:06:24.996622+00:00 app[worker.1]:     at WebSocketConnection.triggerReady (/app/node_modules/discord.js/src/client/websocket/WebSocketConnection.js:129:24)
2021-01-09T19:06:24.996623+00:00 app[worker.1]:     at /app/node_modules/discord.js/src/client/websocket/packets/handlers/Ready.js:64:28
2021-01-09T19:06:24.996623+00:00 app[worker.1]:     at Timeout.<anonymous> (/app/node_modules/discord.js/src/client/Client.js:436:7)
2021-01-09T19:06:24.996712+00:00 app[worker.1]: (node:4) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 131)
2021-01-09T19:06:24.996801+00:00 app[worker.1]: (node:4) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'match' of null
2021-01-09T19:06:24.996802+00:00 app[worker.1]:     at Client.<anonymous> (/app/index.js:131:31)
2021-01-09T19:06:24.996802+00:00 app[worker.1]:     at Client.emit (events.js:315:20)
2021-01-09T19:06:24.996802+00:00 app[worker.1]:     at MessageCreateHandler.handle (/app/node_modules/discord.js/src/client/websocket/packets/handlers/MessageCreate.js:9:34)
2021-01-09T19:06:24.996803+00:00 app[worker.1]:     at WebSocketPacketManager.handle (/app/node_modules/discord.js/src/client/websocket/packets/WebSocketPacketManager.js:108:65)
2021-01-09T19:06:24.996803+00:00 app[worker.1]:     at /app/node_modules/discord.js/src/client/websocket/packets/WebSocketPacketManager.js:73:12
2021-01-09T19:06:24.996804+00:00 app[worker.1]:     at Array.forEach (<anonymous>)
```

Bot is not working with latest update. the log repeats itself infinitely often that my browser even hangs up

**Comment by GiorgioBrux** [comment]
Can you post the full log?

**Comment by GiorgioBrux** [comment]
Nevermind, found it.
It's a problem with the new privnote detection (**sigh**). I will fix it in the new release, in the meanwhile feel free to ignore it as the sniper will still snipe correctly.

**Comment by GiorgioBrux** [comment]
Fixed in 1aa80d3, sorry!

**Comment by bonxz** [comment]
I could not copy the log because the error is so long that the window hangs :D
